overlord/snapstate/handlers_mount_test.go

```diff
@@ -44,8 +44,8 @@ type mountSnapSuite struct {
 var _ = Suite(&mountSnapSuite{})
 
 func (s *mountSnapSuite) SetUpTest(c *C) {
-	oldDir := dirs.SnapCookieDir
-	dirs.SnapCookieDir = c.MkDir()
+	oldDir := dirs.GlobalRootDir
```

Comment:
Not sure we need to preserve the old global root dir, we can just reset to `/` all the time. Still, just a nitpick, thank you for cleaning this up :)

Reply:
agreed, i'll nuke this when i'm in the neighbourhood again